src/call.c

```diff
@@ -2504,3 +2509,17 @@ void call_start_answtmr(struct call *call, uint32_t ms)
 
 	tmr_start(&call->tmr_answ, ms, delayed_answer_handler, call);
 }
+
+
+bool call_is_established(struct call *call)
+{
+	return  call ?
+		call_state(call) == CALL_STATE_ESTABLISHED &&
+		!call_is_onhold(call) : false;
+}
```

Comment:
perhaps a better name is `call_is_active` ?

also the input param should be `const`

Reply:
Also a call that is in early state can be onhold and there may be need to be able to test it.  If we introduce a new API call for each combination, we'll end up with 6 new API calls, which does not make sense to me.  It would be cleaner to just have an API call that tells the state and another that tells if the call is (or was) onhold.